backend/spec/factories/survey_subjects.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :survey_subject do
+    name { 'Animals' }
+    description { 'All surveys have questions related to animals.' }
+    initialize_with { SurveySubject.find_or_create_by(name: 'Animals') }
```

Comment:
why is it like this, maybe it should use something from faker, or do you think not?

Reply:
Oh, I did it because I thought it's like question type and role... As only Admin can create this, but I can use faker if preferable.